integration_tests/mockito-kotlin/build.gradle

```diff
@@ -0,0 +1,17 @@
+import org.robolectric.gradle.RoboJavaModulePlugin
+
+apply plugin: RoboJavaModulePlugin
+
+apply plugin: 'org.jetbrains.kotlin.jvm'
+
+dependencies {
+    api project(":robolectric")
+    compileOnly AndroidSdk.MAX_SDK.coordinates
+
+    testCompileOnly AndroidSdk.MAX_SDK.coordinates
+    testRuntimeOnly AndroidSdk.MAX_SDK.coordinates
+    testImplementation("androidx.test.ext:junit:$axtJunitVersion")
+    testImplementation "junit:junit:${junitVersion}"
```

Comment:
Remove {}?

Reply:
do you mean remove () and add {} in line 13? 

> Remove {}?

Reply:
I think this can just be e.g. `androidx.test.ext:junit:$axtJunitVersion`

Reply:
OK, both with or without {} are OK, maybe keeping pace with the usages in other tests' modules is good, too. Aleady changed to `androidx.test.ext:junit:${axtJunitVersion}` for it.

Reply:
OK, I will take the suggestion for this.